package.json

```diff
@@ -6,7 +6,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "webpack-dev-server --config webpack.config.js",
+    "start": "webpack-dev-server --hot --inline --config webpack.config.js",
```

Comment:
Could you move this to the devServer section of webpack.config.js? That way we have all the config in one place